content/docs/user-guide/platforms/linux/_index.md

```diff
@@ -21,8 +21,8 @@ sudo apt-get update
 sudo apt-get install -y \
     build-essential clang-6.0 uuid-dev libz-dev libncurses5-dev libcurl4-openssl-dev \
     mesa-common-dev libglu1-mesa-dev libjpeg-dev libjbig-dev libsdl2-dev \
-    libxcb-xinerama0 libxcb-xinput0 \
-    libopus-dev libwebp-dev
+    libxcb-xinerama0 libxcb-xinput0 libfontconfig1-dev \
+    libopus-dev libwebp-dev 
```

Comment:
An extra space has been added to the end of this line

Reply:
That wasn't me, that was git

```
warning: LF will be replaced by CRLF in content/docs/user-guide/platforms/linux/_index.md.
```